yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerSSHLauncher.java

```diff
@@ -74,7 +74,7 @@ public void appendContainerConfig(DockerSlaveTemplate dockerSlaveTemplate, Creat
             createCmd.withCmd("bash", "-c", "/usr/sbin/sshd -D -p " + sshPort);
         }
 
-        createCmd.getPortBindings().add(PortBinding.parse("0.0.0.0::" + sshPort));
+        createCmd.getPortBindings().add(PortBinding.parse("" + sshPort));
```

Comment:
It is not required, it only states to me that there should be string. I can remove it if it OK for you.